cql3/expr/expression.cc

```diff
@@ -1679,14 +1679,14 @@ cql3::raw_value do_evaluate(const binary_operator& binop, const evaluation_input
 }
 
 // Evaluate a conjunction of elements separated by AND.
-// NULL is treated as an "unkown value" - maybe true maybe false.
```

Comment:
That I'm not sure about - as there used to be a UKNOWN in C* once, but it was renamed at some point to UNKNOWN...